menu/seagl_2023.json

```diff
@@ -0,0 +1,22 @@
+{
+	"version": 2023110100,
+	"url": "https://osem.seagl.org/conferences/seagl2023/schedule.xml",
+	"title": "SeaGL 2023",
+	"start": "2023-11-03",
+	"end": "2023-11-04",
+	"timezone": "US/Pacific",
+	"metadata": {
+		"links": [
+			{
+				"url": "https://seagl.org/",
+				"title": "Site web"
```

Comment:
That sounds a bit more French than what I'm used to in Seattle? (-;

Reply:
oh lol yeah I copied the Capitole file and missed that :-)

Reply:
Fixed